### DISCUSSION TITLE
RPiR Deadlines

### DISCUSSION THREAD

**Discussion by AlexaRoditi** [IBAHCM/RPiR#171] [Archive]
Hi @IBAHCM/rpir-instructors, just a quick question about the course timeline please. Is there a  list of all the deadlines somewhere? Think I've missed it somewhere but if you could send a link, would be helpful thank you!

**Comment by richardreeve** [comment]
The only location at the moment is at the top of the Moodle page (and only covers this week), but the deadlines are intended to be 1pm on Mondays, starting this Monday for the second practical series. I haven't written them down because I wanted to see how things were progressing, but I'll sort that out before the lectures.

**Reply by AlexaRoditi** [reply]
No worries, that's good to know thank you

**Reply by richardreeve** [reply]
Obviously deadlines are now Wed 1pm this week, and we'll aim to make it Wed 1pm next week too.
